pkg/receiver/activedirectoryinvreceiver/config.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package activedirectoryinvreceiver
+
+import (
+	"errors"
+	"regexp"
+	"time"
+)
+
+// ADConfig defines configuration for Active Directory Inventory receiver.
+
+type ADConfig struct {
+	DN           string        `mapstructure:"base_dn"` // DN is the base distinguished name to search from
```

Comment:
Ideally the name of the variable in the config struct would be as close as possible to the configuration property name.
```suggestion
	BaseDN           string        `mapstructure:"base_dn"` // DN is the base distinguished name to search from
```